src/Mod/PartDesign/App/FeatureHole.h

```diff
@@ -83,10 +83,11 @@ class PartDesignExport Hole : public ProfileBased
         const char * designation;
         double diameter;
         double pitch;
+        double CoreHole;

```

Comment:
I can't quite grasp the FreeCAD naming convention, but given that the other members here are lowercase, should this be `coreHole` instead?

Reply:
as I wrote in another PR, you are free, as long as the name makes clear what the variable is about

Reply:
> as I wrote in another PR, you are free, as long as the name makes clear what the variable is about

I don't quite agree. Usually in feature and view provider classes we use for properties (i.e. App::Property and sub-classes) names that start with an uppercase character while local variables preferably should start with a lowercase character. This makes it easier to distinguish between properties and normal variables.

Anyway, I will accept the PR as it is.